.ci/build.sh

```diff
@@ -18,6 +19,7 @@ python manage.py fetch_deployed_data --allow-failure _site $EXPORTED_DATA
 if [[ -n "$GCI_TOKEN" ]]; then
   python manage.py fetch_gci_task_data private
   python manage.py cleanse_gci_task_data private _site
+  python manage.py filter_gci_non_issue_tasks private _site
```

Comment:
This command only has one argument but you provide two here
Anyway, is this necessary? As https://gitter.im/coala/community?at=5bd9afd8cd2a6b5bdc73564b suggests, it does not need to be published.